src/oc_erchef/apps/oc_chef_wm/src/chef_wm_rabbitmq_management.erl

```diff
@@ -169,13 +169,14 @@ parse_max_length_response(Message) ->
     end.
 
 
--spec sync_check_queue_at_capacity(atom(), string(), string()) -> {integer(), integer(), boolean()}.
+-spec sync_check_queue_at_capacity(atom(), string(), string()) -> {integer(), integer(), false}.
 sync_check_queue_at_capacity(PoolNameAtom, Vhost, Queue) ->
     Result = check_current_queue_state(PoolNameAtom, Vhost, Queue, 0),
     case Result of
-      skipped  -> {0, 0, false};
-      {MaxLength, reset_dropped_since_last_check} -> {MaxLength, 0, false};
-      {MaxLength, N, QueueAtCapacity} -> {MaxLength, N, QueueAtCapacity}
+      skipped  -> {0, 0, false}
+      %% rabbitmq is disabled, so Result always = skipped ?
+      %{MaxLength, reset_dropped_since_last_check} -> {MaxLength, 0, false};
+      %{MaxLength, N, QueueAtCapacity} -> {MaxLength, N, QueueAtCapacity}
```

Comment:
Why is this code commented.  I think we fixed the spec on the assumption this had previously been commented, but now I see this was a recent change.  We should sort out what is up here with this uncommented.

Reply:
Fixed.